### ISSUE TITLE
Crash on homepage example command

### ISSUE THREAD

**Issue by carlocab** [spaam/svtplay-dl#1345]
### svtplay-dl versions:

```
❯ svtplay-dl --version
svtplay-dl 3.1
```

`svtplay-dl` was installed via Homebrew.

### Operating system and Python version:

```
❯ sw_vers
ProductName:    macOS
ProductVersion: 11.2.3
BuildVersion:   20D91
❯ python3 --version
Python 3.9.2
```

### What is the issue:

```
❯ svtplay-dl https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden
ERROR: svtplay-dl crashed
ERROR: Run again and add --verbose as an argument, to get more information
ERROR: If the error persists, you can report it at https://github.com/spaam/svtplay-dl/issues
ERROR: Include the URL used, the stack trace and the output of svtplay-dl --version in the issue
❯ svtplay-dl --verbose https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden
DEBUG [1615853809.372328] /usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py/get_media: version: 3.1
DEBUG [1615853809.374464] /usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/service/__init__.py/__init__: service: svtplay
DEBUG [1615853809.374686] /usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden'
DEBUG [1615853809.406037] /usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.svtplay.se:443
DEBUG [1615853810.066429] /usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/urllib3/connectionpool.py/_make_request: https://www.svtplay.se:443 "GET /video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden HTTP/1.1" 404 19742
Traceback (most recent call last):
  File "/usr/local/bin/svtplay-dl", line 7, in <module>
    svtplay_dl.main()
  File "/usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/__init__.py", line 69, in main
    get_media(urls[0], config, __version__)
  File "/usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py", line 61, in get_media
    get_one_media(stream)
  File "/usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py", line 111, in get_one_media
    for i in streams:
  File "/usr/local/Cellar/svtplay-dl/3.1/libexec/lib/python3.9/site-packages/svtplay_dl/service/svtplay.py", line 83, in get
    if not vid and not self.visibleid:
AttributeError: 'Svtplay' object has no attribute 'visibleid'
```

Related: https://github.com/Homebrew/homebrew-core/pull/73013

**Comment by spaam** [comment]
this is fixed in 3.2 but had some other issue so that PR didn't merge. 
made a new release 3.3 and a PR for that https://github.com/Homebrew/homebrew-core/pull/73289
closing this :)

**Comment by carlocab** [comment]
I actually tried installing 3.2 from source using your PR but also saw the same error 😅 

But, okay, I hope it's fixed now!
